README.md

```diff
@@ -118,6 +120,20 @@ The **API endpoint** can be used to:
   scenarios; applies a cgroup/namespace isolation barrier and then
   drops privileges.
 
+## Supported platforms
+
+We continuously test Firecracker on machines with the following CPUs micro-architectures:
```

Comment:
Nit: I think an in-line list would work better since each bullet point has just a few chars.

Reply:
done

Reply:
Bullet point in here is more readable IMO in this case and easier to maintain in long run.

Comment from a user which is closely following the project and reads docs ;)

Reply:
Thanks @gregorybreski. The bullet points really help out with a long list of CPUs, but currently it is short. Since this was a nit from @raduweiss , I will revert probably.